### PR TITLE
refactor: [#186982853] add elevator non essential question behind feature flag

### PR DESCRIPTION
**Pull request by SirSamuelJoseph**
## Description
Setup non-essential question for elevator-owners and a corresponding task to be added to the roadmap

### Ticket

[186982853](https://www.pivotaltracker.com/story/show/186982853)

### Notes
The "elevator_registration" task is still in review, so all of this content is behind a feature flag until that is merged into main

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: api/serverless.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ const dynamicsElevatorSafetyURL = process.env.DYNAMICS_ELEVATOR_SAFETY_URL || ""
 const dynamicsElevatorSafetyClientId = process.env.DYNAMICS_ELEVATOR_SAFETY_CLIENT_ID || "";
 const dynamicsElevatorSafetySecret = process.env.DYNAMICS_ELEVATOR_SAFETY_SECRET || "";
 const dynamicsElevatorSafetyTenantId = process.env.DYNAMICS_ELEVATOR_SAFETY_TENANT_ID || "";
+const featureElevatorOwningBusiness = process.env.FEATURE_ELEVATOR_OWNING_BUSINESS || "";
```
Comment (SirSamuelJoseph) [inline api/serverless.ts:75]: Added this feature flag but didn't update any environmental files, because it's all hidden until we get the task approved

Comment (FarazA22) [reply api/serverless.ts:75]: you may have already done this, but if you haven't, you should add the variable to bitwarden.

File: content/src/fieldConfig/profile.json [hunk 2]
```diff
@@ -240,6 +240,15 @@
           "altDescription": "Do you plan to be a `home-based business|home-based-business` ?"
         }
       },
+      "elevatorOwningBusiness": {
+        "default": {
+          "header": "Elevator Owning Business",
+          "radioButtonTrueText": "Yes",
+          "radioButtonFalseText": "No",
+          "description": "Do you own a commercial or manufacturing space with an `elevator-device|elevator-device` ?",
+          "altDescription": "Do you plan to own a commercial or manufacturing space with an `elevator-device|elevator-device` ?"
```
Comment (FarazA22) [inline content/src/fieldConfig/profile.json:249]: dust - the altDescription is using the same copy, is it needed?

Comment (SirSamuelJoseph) [reply content/src/fieldConfig/profile.json:249]: It actually isn't the same copy, the alt description talks about "plans to own" and the regular description is around "currently owns"

File: web/src/pages/profile.tsx [hunk 3]
```diff
@@ -543,6 +550,16 @@ const ProfilePage = (props: Props): ReactElement => {
         >
           <HomeBasedBusiness />
         </ProfileField>
+
+        <ProfileField
+          fieldName="elevatorOwningBusiness"
+          displayAltDescription={displayAltHomeBasedBusinessDescription}
```
Comment (FarazA22) [inline web/src/pages/profile.tsx:556]: dust - if we are displaying `displayAltHomeBasedBusinessDescription` why are we storing description  and altDescription?

**Review by adidner — APPROVED** [review]
Approved, with a few dusty question / comments

File: web/src/lib/roadmap/buildUserRoadmap.ts [hunk 4]
```diff
@@ -151,6 +151,10 @@ const getIndustryBasedAddOns = (profileData: ProfileData, industryId: string | u
     addOns.push("logistics-modification");
   }
 
+  if (profileData.elevatorOwningBusiness && process.env.FEATURE_ELEVATOR_OWNING_BUSINESS) {
+    addOns.push("elevator-registration");
+  }
```
[the remark this answers is not in the document]
Comment (SirSamuelJoseph) [reply web/src/lib/roadmap/buildUserRoadmap.ts:156]: added!

File: web/src/pages/profile.tsx (hunk 3 above)
Comment (SirSamuelJoseph) [reply web/src/pages/profile.tsx:566]: `displayAltHomeBasedBusinessDescription` determines whether or not the alt description is served

**Pull request merged**